### DISCUSSION TITLE
9 by 16 (1080 px by 1920 px) Aspect Ratio.

### DISCUSSION THREAD

**Discussion by chope1** [lllyasviel/Fooocus#3374] [Q&A] [answered]
I checked the image dimensions and couldn't find a very popular 1080 px by 1920 px or 9 by 16 aspect ratio.
Knowing that it's the most popular, is there any reason it wasn't included?

You guys did a very good job here.
By far the best AI  text to image generator.
Thanks.

**Comment by mashb1t** [comment, marked as answer]
Yes, you can read all about why here: https://github.com/lllyasviel/Fooocus/pull/1617

**Reply by chope1** [reply]
Thanks for your response, unfortunately, I can't see it in the options.
Below is my screenshot -

Note: I am using the Colab hosted option using the Gradio.live provided link.
![dimension](https://github.com/user-attachments/assets/cb0d75e7-795b-4e26-8533-cde3d29b81b1)

**Reply by mashb1t** [reply]
> is there any reason it wasn't included

The resolution does not exist in Fooocus, reasons see in the link provided in my last comment.
You can manually add any resolutions you like using the config.txt, see config_modification_tutorial.txt, but i highly advise against it if you value image quality and consistency / prompt adherence.
